### PR TITLE
Add module for Analytics Dashboard

### PR DESCRIPTION
**Pull request by Savidude**

### CODE REVIEW

File: modules/apim_analytics_dashboard/manifests/init.pp [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# ----------------------------------------------------------------------------
+#  Copyright (c) 2018 WSO2, Inc. http://www.wso2.org
+#
```
Comment (nihlaakram) [inline modules/apim_analytics_dashboard/manifests/init.pp:3]: Copy right year should be changed to 2019

File: modules/apim_analytics_dashboard/manifests/custom.pp [hunk 2]
```diff
@@ -14,19 +14,8 @@
...
-    path    => "/bin/",
-  }
-
-  # Start the service
-  service { $profile:
-    enable => true,
-    ensure => running,
-    subscribe => File["wso2-binary"],
-  }
+# Class: apim_analytics_dashboard::custom
```
Comment (VimukthiPerera) [inline modules/apim_analytics_dashboard/manifests/custom.pp:17]: Why is this file removed from the worker?

File: modules/apim_analytics_dashboard/manifests/init.pp (hunk 1 above)
Comment (Savidude) [reply modules/apim_analytics_dashboard/manifests/init.pp:3]: The analytics module is being reintroduced after being removed. The content is the same as what was once there earlier.

**Review by VimukthiPerera — APPROVED** [review]
